archiver_test.go

```diff
@@ -74,7 +74,7 @@ func symmetricTest(t *testing.T, name string, ar Archiver) {
 
 		if info.IsDir() {
 			// stat dir instead of read file
-			_, err := os.Stat(origPath)
+			_, err = os.Stat(origPath)
```

Comment:
I think it depends largely on the context. That might be why go vet doesn't issue warnings about it...